scripts/tests/index.js

```diff
@@ -33,7 +32,7 @@ global.deepEqual = (x, y) => {
  * @return {Boolean}
  */
 global.compare = (a, b) => {
-	return a.innerHTML === b.replace(/[\n\t ]/g, '');
+	return a.innerHTML === b.replace(/\n|\t|  +/g, '');
```

Comment:
== `\s+`?

Reply:
maintains single spaces in something like `<a href="#">`.

Reply:
I see, I missed that it was 2 spaces
would be clearer with `\s{2,}`